cms-docker-gh-issues.py

```diff
@@ -91,20 +92,22 @@
         pulls_obj = json.loads(pulls_obj)
         urls = ""
         for pull in pulls_obj:
-            urls += str(pull["html_url"]) + " "
-        print("The following PRs have matching labels: ", urls)
+            pull_obj = pull.get("pull_request")
+            if pull_obj != None:
+                urls += "* " + str(pull_obj.get("html_url")) + "\n"
+        print("The following PRs have matching labels: \n", urls)
 
-        # Get current issue number
-        print("Check newly created Issue", issues_curl)
-        exit_code, issues_obj = run_cmd(issues_curl)
-        issues_dict = json.loads(issues_obj)
-        issue_number = issues_dict["items"][0]["number"]
+        issues = gh_repo.get_issues(labels=[str(label) for label in args.labels])
+        issue_number = issue.number
```

Comment:
@aandvalenzuela , this will fail as there is no `issue`. For this section why not use the issue returned by `gh_repo.create_issue` call?